index.bs

```diff
@@ -5157,36 +5157,36 @@ In order to perform a [=registration ceremony=], the [=[RP]=] MUST proceed as fo
     with the following contents:
 
     <dl>
-        :   [=credential record/type=]
+        :   [$credential record/type$]
         ::  <code>|credential|.{{Credential/type}}</code>.
 
-        :   [=credential record/id=]
+        :   [$credential record/id$]
         ::  <code>|credential|.{{Credential/id}}</code> or <code>|credential|.{{PublicKeyCredential/rawId}}</code>,
             whichever format is preferred by the [=[RP]=].
 
-        :   [=credential record/publicKey=]
+        :   [$credential record/publicKey$]
         ::  The [=credential public key=] in |authData|.
 
-        :   [=credential record/signCount=]
+        :   [$credential record/signCount$]
         ::  <code>|authData|.[=authData/signCount=]</code>.
 
-        :   [=credential record/transports=]
+        :   [$credential record/transports$]
         ::  The value returned from <code>|response|.{{AuthenticatorAttestationResponse/getTransports()}}</code>.
 
-        :   [=credential record/BE=]
+        :   [$credential record/BE$]
         ::  The value of the [=authData/flags/BE=] [=flag=] in |authData|.
 
-        :   [=credential record/BS=]
+        :   [$credential record/BS$]
```

Comment:
Any reason not to make this a bit more readable and call the column `backupStatus` instead? It'll save people a click who don't know to equate "BS" to the flag's full name.

Reply:
Good point, thanks! But unrelated to this change, so let's tackle that in issue #1811.